iothub/device/tests/RetryDelegatingHandlerImplicitOpenTests.cs

```diff
@@ -83,14 +83,15 @@ public async Task Retry_CloseAsyncSuccessDisposesHandler_Throws()
         {
             var contextMock = Substitute.For<IPipelineContext>();
             var innerHandlerMock = Substitute.For<IDelegatingHandler>();
-            innerHandlerMock.OpenAsync(CancellationToken.None).Returns(t => TaskHelpers.CompletedTask);
-            innerHandlerMock.CloseAsync(CancellationToken.None).Returns(t => TaskHelpers.CompletedTask);
+            var cancellationToken = new CancellationToken();
+
+            innerHandlerMock.OpenAsync(cancellationToken).Returns(t => TaskHelpers.CompletedTask);
+            innerHandlerMock.CloseAsync(cancellationToken).Returns(t => TaskHelpers.CompletedTask);
 
             var sut = new RetryDelegatingHandler(contextMock, innerHandlerMock);
-            await sut.OpenAsync(CancellationToken.None).ConfigureAwait(false);
-            await sut.CloseAsync(CancellationToken.None).ConfigureAwait(false);
+            await sut.OpenAsync(cancellationToken).ConfigureAwait(false);
+            await sut.CloseAsync(cancellationToken).ConfigureAwait(false);
 
-            var cancellationToken = new CancellationToken();
             await ((Func<Task>)(() => sut.OpenAsync(cancellationToken))).ExpectedAsync<ObjectDisposedException>().ConfigureAwait(false);
```

Comment:
Previously we were trying to call the mock with a new object. Trying to use the same object so that the test definitely uses the correct mock.